src/foam/mlang/mlang.js

```diff
@@ -317,7 +317,8 @@ foam.CLASS({
       return foam.String.constantize(this.cls_.name) + '(' +
           this.arg1.toString() + ', ' +
           this.arg2.toString() + ')';
-    }
+    },
+    function contradicts(other) { return false; }
```

Comment:
I'm open to ideas. I'm worried that in the "do not change" case, we may make expressions more complex; e.g., if `a.comparisonMethodOfSomeKind(b)` wishes to return an equivalent mLang when embedded in an `AND`, we may get: `AND(a, b, c) ==> a.comparisonMethodOfSomeKind(b) ==>  AND(AND(a, b), c)`